Experiment scripts for frequency sweep

dianarg opened the pull request:
- Frequency sweep helpers and run script for DGEMM
- Runtime/energy vs frequency plot based on frequency sweep

dianarg committed:
Add frequency_sweep scripts

Signed-off-by: Diana Guttman <diana.r.guttman@intel.com>

bgeltz commented on integration/apps/geopmbench.py:
```diff
@@ -0,0 +1,76 @@
+import os
```
Missing license header.

bgeltz commented on integration/apps/geopmbench.py:
```diff
@@ -0,0 +1,76 @@
+import os
+import textwrap
+
+from . import apps
```
Why not just ```import apps```?

dianarg replied:
That pattern didn't work unless CWD was in the PYTHONPATH.  We probably need to look through all the files and make sure we import in a consistent way.  There are also some differences between python2 and 3.

bgeltz reviewed (commented):
I think I'd benefit from a review session on all of this.  I didn't realize there was so much new code.

dianarg commented:
This PR is one patch on top of #1233

dianarg committed:
Clean up runtime energy plot

Signed-off-by: Diana Guttman <diana.r.guttman@intel.com>

dianarg committed:
Add script to find best-fit frequency map

Signed-off-by: Diana Guttman <diana.r.guttman@intel.com>

cmcantalupo reviewed (dismissed):
LGTM.

bgeltz committed:
Add per-region summary script

- Generates 2 files:
  + region_stats.log - Full stats for each region/freq
  + region_mean_stats - Mean stats for each region/freq
- Also generates summaries for Epoch and application totals.

Change-Id: I9a65b23967c8867c2d8cfa0bf5b7bac9ada30bb0
Signed-off-by: Brad Geltz <brad.geltz@intel.com>
